zebra-rpc/src/methods.rs

```diff
@@ -601,6 +656,14 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 #[derive(Copy, Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
 
+#[derive(serde::Serialize)]
+/// Response to a `z_gettreestate` RPC request.
+///
+/// Contains the hex-encoded Sapling & Orchard note commitment trees.
+// TODO: adjust the description above so that it reflects the new fields once
+// they are added.
+pub struct GetTreestate(sapling::tree::NoteCommitmentTree);
```

Comment:
Nit: we usually derive all the traits we might need, right before the struct declaration.

```suggestion
/// Response to a `z_gettreestate` RPC request.
///
/// Contains the hex-encoded Sapling & Orchard note commitment trees.
// TODO: adjust the description above so that it reflects the new fields once
// they are added.
#[derive(Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
pub struct GetTreestate(sapling::tree::NoteCommitmentTree);
```

Reply:
Thanks, I wanted to be consistent with the other structs in the file. They had the `derive` attributes above the doc comments. It seemed unfamiliar.

Reply:
We changed to the standard format as part of PR #3143.